### ISSUE TITLE
[FEATURE REQUEST] Feroxbuster doesn't give an option to change Scan Limit using Scan Management Menu

### ISSUE THREAD

**Issue by sectroyer** [epi052/feroxbuster#817]
**Describe the bug**
Once application is launched there is no way to modify Scan Limit using Scan Management Menu.

**Comment by epi052** [comment]
Agreed, this would be a nice addition! I'd happily help someone work through a PR to make this happen. Thanks for taking the time to submit this!
